ompi/mpi/fortran/mpiext-use-mpi/Makefile.am

```diff
@@ -21,6 +21,7 @@ if OMPI_BUILD_FORTRAN_USEMPI_OR_USEMPIF08_EXT
 
 AM_FCFLAGS = -I$(top_builddir)/ompi/include -I$(top_srcdir)/ompi/include \
              $(OMPI_FC_MODULE_FLAG)$(top_builddir)/ompi/mpi/fortran/base \
+             $(OMPI_FC_MODULE_FLAG)$(top_builddir)/ompi/mpi/fortran/use-mpi-ignore-tkr \
```

Comment:
Is this always guaranteed to be `use-mpi-ignore-tkr`?  I.e., is it possible for OMPI to be building the `use-mpi-tkr` in the same build as Open MPI extensions with Fortran `use mpi` bindings?

Reply:
Yes, it is guaranteed. Fortran bindings of MPI Extensions are not enabled for `use-mpi-tkr`. You wrote the code and comment :)
https://github.com/open-mpi/ompi/blob/master/config/ompi_ext.m4#L182